plugins/inputs/ipmi_sensor/connection.go

```diff
@@ -15,11 +15,13 @@ type Connection struct {
 	Port      int
 	Interface string
 	Privilege string
+	HexKey    string
 }
 
-func NewConnection(server string, privilege string) *Connection {
+func NewConnection(server, privilege, hexKey string) *Connection {
 	conn := &Connection{}
 	conn.Privilege = privilege
+	conn.HexKey = hexKey
```

Comment:
You can use
```suggestion
	conn := &Connection{
		Privilege: privilege,
		HexKey: hexKey,
	}
```
for initialization.

Reply:
Good point, missed that.